components/gc-subway/includes/docs-questions.html

```diff
@@ -0,0 +1,9 @@
+<ul class="list-inline">
```

Comment:
No sure if this file fit under "includes" neither under "samples". But it do help to maintain repetition on the sub-pages. Let's think where it will be fit best, for sure not in the "includes" folder.

Reply:
What about "fragments" folder and use "include_relative"